### ISSUE TITLE
Certified Nodes tracking in explorer

### ISSUE THREAD

**Issue by andhartl** [threefoldtech/tfexplorer#341]
a certified node field needs to be implemented in explorer to track cert nodes for minting and show which capacity is coming from cert nodes. needs to be implemented at least week before we mint for august.

**Comment by muhamadazmy** [comment]
@LeeSmet we already have the flag im explorer. Is this enough for the minting modifications? Or requires more changes ?

**Comment by weynandkuijpers** [comment]
## The need

For the August minting period we need to have the notion implemented in the current minting process to recognize certified and non-certified 3nodes.  The necessity for this comes from the fact that 1000+ titanv2 node shave been sold in a recent  selling spree.

## Definitions

Certfied nodes are node that meet the following criteria:
- a titanV2 node sold by better token with a locked up bios and a description stored in Bettertoken: https://circles.threefold.me/project/funnel_diy_farming/kanban
- a node that is owned, installed and operated by a certified farmer who has a signed contract about the obligations to comply with as a certified farmers.
- Certification in V2 of the minting process is tracked by nodes, not by farms.
- Node certification needs to be approved by the guardians who are also involved in the minting 

## Requirements
- the node registry (the explorer) needs to have the ability to make a 3node as certified or non-certified
- for minting purposes the minting algorithm needs to recognize a diffe
- rent number of TFT's per CU/SU/NU based on the certified flag.
- 
- 
## Proposal
- create an additional field in the registry / explorer to record the 3node type: certified or DIY
- create a process for the minting guardians to agree the 3node type
      -  all titan V2 3nodes are certified
      - all (professional) farmers that have signed the certification contract have certified nodes.  The actual process of certification (closing down the BIOS and locking the boot mechanism to a known  zero-OS version is to be done at a future point in time.

## Timeline
- changes to the 3node registry and explorer need to be done before the August minting process (1-3 September)
- changes to the minting algorithm need to be done before the August minting process (1-3 September)

Grid V3 will make supersede this soon after, but we cannot not provide certified minting to the new TitanV2 owners.

**Comment by andhartl** [comment]
@LeeSmet as army wrote: is there already a certified flag/field we can use for august minting? when will the v3 explorer be ready for production? The reason I am asking is that if its sept for production we might go manual for august. of course with the blessing go the wisdom council.

**Comment by LeeSmet** [comment]
we can use whatever flag, as long as we understand it is only used to mark certified nodes. Iirc there is an "approved" flag or smthing which might not be used yet, we can use that then. This has as benefit that the minting aggregator won't need to be updated. But again, need to see if the flag is for sure not used yet then,

Even if the flag is there, the minting code will ofcourse need to be adjusted to account for it when calculating the actual payouts

**Comment by LeeSmet** [comment]
No longer relevant with move to grid 3
